### PR TITLE
Title

### PR DESCRIPTION
**Pull request by houghj16**
## Description
- item 1
- item 2

### CODE REVIEW

File: ShareX.HelpersLib/Extensions/Extensions.cs [hunk 1]
```diff
@@ -32,7 +32,6 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
-using System.Reflection;
```
Comment (houghj16) [inline ShareX.HelpersLib/Extensions/Extensions.cs:35]: Comment1

File: ShareX.ScreenCaptureLib/Forms/FFmpegOptionsForm.Designer.cs [hunk 2]
```diff
@@ -70,19 +70,20 @@ private void InitializeComponent()
             this.tpGIF = new System.Windows.Forms.TabPage();
             this.lblGIFDither = new System.Windows.Forms.Label();
             this.lblGIFStatsMode = new System.Windows.Forms.Label();
-            this.tbAMF = new System.Windows.Forms.TabPage();
+            this.tpAMF = new System.Windows.Forms.TabPage();
+            this.lblAMFBitrateK = new System.Windows.Forms.Label();
```
Comment (houghj16) [inline ShareX.ScreenCaptureLib/Forms/FFmpegOptionsForm.Designer.cs:74]: comment2

File: ShareX.HelpersLib/ShareX.HelpersLib.csproj [hunk 3]
```diff
@@ -150,6 +150,7 @@
     </Compile>
     <Compile Include="Cryptographic\DPAPI.cs" />
     <Compile Include="Emoji.cs" />
+    <Compile Include="Extensions\FormExtensions.cs" />
```
Comment (otheruser16) [inline ShareX.HelpersLib/ShareX.HelpersLib.csproj:153]: single comment

File: ShareX.HelpersLib/Extensions/Extensions.cs [hunk 4]
```diff
@@ -32,7 +32,6 @@
...
 using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Forms;
```
Comment (houghj16) [inline ShareX.HelpersLib/Extensions/Extensions.cs:37]: new comment

**Pull request closed without merging**